src/Core/Models/Business/SubscriptionCreateOptions.cs

```diff
@@ -5,7 +5,7 @@ namespace Bit.Core.Models.Business;
 
 public class OrganizationSubscriptionOptionsBase : Stripe.SubscriptionCreateOptions
 {
-    public OrganizationSubscriptionOptionsBase(Organization org, StaticStore.Plan plan, TaxInfo taxInfo, int additionalSeats, int additionalStorageGb, bool premiumAccessAddon)
+    public OrganizationSubscriptionOptionsBase(ISubscriber org, StaticStore.Plan plan, TaxInfo taxInfo, int additionalSeats, int additionalStorageGb, bool premiumAccessAddon)
```

Comment:
Sorry, I didn't mean that you have to use `ISubscriber` here in the public interfaces. These methods won't take any object that implements this interface, it does specifically need an `Organization`. So it's fine to specify `Organization` here.

I only meant that you should use `org.SubscriberName()` method to get the subscriber name, rather than accessing `org.Name` directly.